### PR TITLE
plan: The stats of some table is outdated, we use pseudo to estimate.

### PR DESCRIPTION
**Pull request by hanfei1991**
@winoros @lamxTyler PTAL

### CODE REVIEW

File: plan/logical_plan_builder.go [hunk 1]
```diff
@@ -1586,12 +1590,24 @@ func (b *planBuilder) buildDataSource(tn *ast.TableName) LogicalPlan {
...
 		// When the first session is created, the handle hasn't been initialized.
-		statisticTable = statistics.PseudoTable(tableInfo.ID)
+		statsTbl = statistics.PseudoTable(tableInfo.ID)
 	} else {
-		statisticTable = handle.GetTableStats(tableInfo.ID)
+		statsTbl = handle.GetTableStats(tableInfo.ID)
+		// TODO: We should consider to add it to metric.
+		if statsTbl.Count == 0 || float64(statsTbl.ModifyCount)/float64(statsTbl.Count) > RatioOfPseudoEstimate {
+			originCnt := statsTbl.Count
+			statsTbl = statistics.PseudoTable(tableInfo.ID)
```
[the remark this answers is not in the document]
Comment (hanfei1991) [reply plan/logical_plan_builder.go]: If so, we need to change a lot of tests.

File: statistics/selectivity.go [hunk 2]
```diff
@@ -84,14 +84,14 @@ func (t *Table) Selectivity(ctx context.Context, exprs []expression.Expression)
 	if t.Count == 0 {
 		return 1, nil
 	}
+	if len(exprs) == 0 {
```
[the remark this answers is not in the document]
Comment (hanfei1991) [reply statistics/selectivity.go]: If it's a pseudo table, we will always return 0.8, but if the condition is empty, we should return 1.

**Comment by hanfei1991** [comment]
@shenli PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**